### DISCUSSION TITLE
where to find the python tutorial?

### DISCUSSION THREAD

**Discussion by zhang-qiang-github** [google-deepmind/mujoco#805] [Asking for Help]
I find a python tutorial : https://github.com/deepmind/mujoco/blob/main/python/tutorial.ipynb

However, all the `xml` file is empty, and I cannot run the example.

Is there any tutorial which can be directly run?

Many thanks.

**Comment by yuvaltassa** [comment]
Did you click the "open in colab" button?

**Reply by zhang-qiang-github** [reply]
Sorry, I cannot open the website from the `open in colab` button. It may be my reason. 

I will try to open the website from the `open in colab` button.

**Reply by yuvaltassa** [reply]
Also, for future reference:

https://mujoco.readthedocs.io/en/latest/python.html#tutorial-notebook
